### DISCUSSION TITLE
Markdown Editor should be WYSIWYG

### DISCUSSION THREAD

**Discussion by Montago** [community/community#5445] [Other Feature Feedback, Questions, & Ideas]
I wish that the Readme MarkDown editor would incorporate a WYSIWYG editor exactly like this: 
https://stackedit.io/app#

Even though i'm a 10+ year professional developer, Markdown is NOT my preferred language to write anything in and i only do so occasionally. The same fact applies to all developers in my team (company even..). 

Documentation is important, and legibility is too. 

Making it easy, makes it more likely that people submit a well formed readme for the next poor soul that needs to Clone a project. 

Thanks !

**Comment by ghost** [comment]
@Montago Hi! I had the same idea as you, but I noticed that you had thought about it before. reference: https://github.com/github/feedback/discussions/17026
